.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, release/* ]
+  pull_request:
+    branches: [ develop ]
+  schedule:
+    - cron: '22 15 * * 2'
```

Comment:
Is this just an arbitrary time of the day?

Reply:
Yes, default provide by Github :)

Reply:
And because things are never random https://en.wikipedia.org/wiki/Patch_Tuesday ;)